e2e-tests/development-runtime/cypress/integration/navigation/redirect.js

```diff
@@ -22,6 +22,69 @@ const runTests = () => {
 
     cy.location(`pathname`).should(`equal`, `/pt/redirect-me/`)
   })
+
```

Comment:
There are two code paths for the cases of:

- directly going to the URL
- clicking a `<Link>` and then getting redirected

So that's why the tests are testing both things